openapi-diff/src/core/OpenApiDiff.Core/Logging/ObjectPath.cs

```diff
@@ -44,20 +58,25 @@ private static ObjectPath ParseRef(string s)
 
         private static JToken FromObject(JObject o, string name)
         {
+            if (name == null)
+            {
+                return null;
+            }
             var @ref = o["$ref"];
             var unrefed = @ref != null ? ParseRef(@ref.Value<string>()).CompletePath(o.Root).Last().token : o;
             return unrefed[name];
         }
 
         private static IEnumerable<(JToken token, string name)> CompletePath(IEnumerable<Func<JToken, string>> p, JToken t)
-            => p.Select(v => {
-                var name = v(t);
-                t =
-                    t is JArray a ? int.TryParse(name, out var i) ? a[i] : null :
-                    t is JObject o ? FromObject(o, name) :
-                    null;
-                return (t, name);
-            });
+            => new[] { (t, "#") }
```

Comment:
seems that we will return something more here than before, `{t}#` extra , what would be a before and after sample return value

Reply:
It's a root property, we needed.